docs/development/virus_scanning.md

```diff
@@ -0,0 +1,53 @@
+# Virus scanning
+
+We use ClamAV to scan for viruses in uploaded bulk submission files. Regardless of if a file is found to contain a virus or not the result of the scan is recorded in the database for auditing purposes.
+
+## Configuration
+
+### Hosted enviroments (uat, staging and production)
+In production we use a container within each App kubernetes pod that has a docker image with clamav installed. The web App pod has clamav "client", `clamav-daemon`, installed via the Dockerfile. There is a configuration file `clamd.container.conf` that opens the clamav container's port, 3310, on localhost, thus allowing the "client" to communicate with the container.
```

Comment:
Not sure this is true? The docker compose file implies it's in a separate pod

Reply:
let's catchup on this.